src/oemof_tabular_plugins/wefe/facades/apv.py

```diff
@@ -250,7 +252,9 @@ def apv_geometry(latitude, y, **kwargs):
             tilt = max(round(abs(latitude)), min_tilt)
             # minimum solar noon altitude (solar angle at solstice when sun is straight south (lat>0) or north (lat<0)
             # source: https://doi.org/10.1016/B978-0-12-397270-5.00002-9
-            min_solar_angle = 90 - round(abs(latitude)) - 23.5
+            angle1 = 90
```

Comment:
what should this angle description's be? i.e. "angle between horizontal and sun at noon" or something like that